content/browser/webui/brave_shared_resources_data_source.cc

```diff
@@ -162,8 +167,12 @@ BraveSharedResourcesDataSource::GetAccessControlAllowOriginForOrigin(
   // back.
   std::string allowed_origin_prefix = content::kChromeUIScheme;
   allowed_origin_prefix += "://";
+  std::string allowed_untrusted_origin_prefix =
+      content::kChromeUIUntrustedScheme;
+  allowed_untrusted_origin_prefix += "://";
   if (!base::StartsWith(origin, allowed_origin_prefix,
-                        base::CompareCase::SENSITIVE)) {
+                        base::CompareCase::SENSITIVE) ||
+      base::StartsWith(origin, allowed_untrusted_origin_prefix)) {
```

Comment:
IIUC, this is actually disallowing access for `chrome-untrusted://` URLs, not allowing? should this be renamed to `disallowed_untrusted_origin_prefix` or simply `untrusted_origin_prefix` then?

Reply:
oh actually I meant to add the `!` and make it `&&`. It should be returning the current origin if the protocol is `chrome` or `chrome-untrusted`. In other words, `{chrome,chrome-untrusted}://brave-resources/*` should be loadable by anything at `chrome://` or `chrome-untrusted://`

Reply:
I fixed that now